totalRP3/core/impl/CommunicationProtocol.lua

```diff
@@ -59,6 +60,9 @@ local PRIORITIES = {
 	HIGH = "HIGH",
 }
 
+local validChannels = {"PARTY", "RAID", "GUILD", "BATTLEGROUND", "WHISPER", "CHANNEL"};
```

Comment:
"Constants" should be declared using UPPERCASE_SNAKE_CASE so we kind of know we should touch them